### ISSUE TITLE
New instance migration error

### ISSUE THREAD

**Issue by TheoCouss** [AzuraCast/AzuraCast#6509]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### Current AzuraCast Version

Rolling Release

### What happened?

Our new instance crashes as startup, it does with both stable and latest version

### Relevant log output

```shell
Starting up AzuraCast services...
Services started up and ready!

AzuraCast Initializing...
=========================

Running Database Migrations
---------------------------


Database Migrations
===================


 [OK] Metadata storage synchronized                                             

Backing up initial database state...
------------------------------------

Running database migrations...
------------------------------

mariadb stdout | 2023-08-16 10:42:25 17 [Warning] Aborted connection 17 to db: 'azuracast' user: 'azuracast' host: 'localhost' (Got an error reading communication packets)
mariadb stdout | 
2023-08-16 10:42:26,598 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
 [ERROR] Database migration failed: An exception occurred while executing a     
         query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table  
         'cache_items' already exists                                           

Attempting to roll back to previous database state...
-----------------------------------------------------

 [WARNING] Your database was restored due to a failed migration.                
                                                                                
           Please report this bug to our developers.                            


Generating Database Proxy Classes
---------------------------------

 Processing entity "App\Entity\Settings"
 Processing entity "App\Entity\SongHistory"
 Processing entity "App\Entity\StorageLocation"
 Processing entity "App\Entity\StationMedia"
 Processing entity "App\Entity\Analytics"
 Processing entity "App\Entity\StationQueue"
 Processing entity "App\Entity\StationHlsStream"
 Processing entity "App\Entity\StationStreamer"
 Processing entity "App\Entity\UserLoginToken"
 Processing entity "App\Entity\StationSchedule"
 Processing entity "App\Entity\StationRemote"
 Processing entity "App\Entity\StationMount"
 Processing entity "App\Entity\Listener"
 Processing entity "App\Entity\StationRequest"
 Processing entity "App\Entity\StationPlaylist"
 Processing entity "App\Entity\StationStreamerBroadcast"
 Processing entity "App\Entity\StationPlaylistFolder"
 Processing entity "App\Entity\UnprocessableMedia"
 Processing entity "App\Entity\Role"
 Processing entity "App\Entity\PodcastCategory"
 Processing entity "App\Entity\Relay"
 Processing entity "App\Entity\AuditLog"
 Processing entity "App\Entity\CacheItem"
 Processing entity "App\Entity\RolePermission"
 Processing entity "App\Entity\StationMediaCustomField"
 Processing entity "App\Entity\SftpUser"
 Processing entity "App\Entity\User"
 Processing entity "App\Entity\ApiKey"
 Processing entity "App\Entity\PodcastMedia"
 Processing entity "App\Entity\StationPlaylistMedia"
 Processing entity "App\Entity\Station"
 Processing entity "App\Entity\StationWebhook"
 Processing entity "App\Entity\CustomField"
 Processing entity "App\Entity\PodcastEpisode"
 Processing entity "App\Entity\Podcast"


 Proxy classes generated to "/var/azuracast/www_tmp/proxies"
Reload System Data
------------------

[2023-08-16T10:42:36.756703+00:00] AzuraCast.ERROR: Doctrine\DBAL\Exception\InvalidFieldNameException: An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_identifier' in 'field list' (uncaught exception) at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 67 while running console command `azuracast:setup` {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 1054): An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_identifier' in 'field list' at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php:67)\n[previous exception] [object] (Doctrine\\DBAL\\Driver\\PDO\\Exception(code: 1054): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_identifier' in 'field list' at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/PDO/Exception.php:28)\n[previous exception] [object] (PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_identifier' in 'field list' at /var/azuracast/www/vendor/doctrine/dbal/src/Driver/PDO/Connection.php:70)"} []
[2023-08-16T10:42:36.757885+00:00] AzuraCast.WARNING: Console command `azuracast:setup` exited with error code 1054. [] []

In ExceptionConverter.php line 67:
                                                                               
  An exception occurred while executing a query: SQLSTATE[42S22]: Column not   
  found: 1054 Unknown column 't0.app_unique_identifier' in 'field list'        
                                                                               

In Exception.php line 28:
                                                                               
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_ident  
  ifier' in 'field list'                                                       
                                                                               

In Connection.php line 70:
                                                                               
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 't0.app_unique_ident  
  ifier' in 'field list'                                                       
                                                                               

azuracast:setup [--update] [--load-fixtures] [--release] [--init]

2023-08-16 10:42:36,766 WARN exited: php-fpm (exit status 1; not expected)
2023-08-16 10:42:37,769 INFO spawned: 'php-fpm' with pid 495
2023/08/16 10:42:37 Command finished successfully.
Starting up AzuraCast services...
Services started up and ready!

AzuraCast Initializing...
=========================

Running Database Migrations
---------------------------


Database Migrations
===================


 [OK] Metadata storage synchronized                                             

Backing up initial database state...
------------------------------------

Running database migrations...
------------------------------

mariadb stdout | 2023-08-16 10:42:37 21 [Warning] Aborted connection 21 to db: 'azuracast' user: 'azuracast' host: 'localhost' (Got an error reading communication packets)
mariadb stdout | 
2023-08-16 10:42:39,085 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
```

**Comment by loan-talvat** [comment]
Same here

**Comment by Blat4** [comment]
Me too

**Comment by blindndangerous** [comment]
Me as well. Except without the missing table, just communication packets.

**Comment by blindndangerous** [comment]
I was able to get mine back up, but it took a bit.  I didn't have any backups, so if you do, I'd use them.  However, my station is small and takes me not long to bring back from nothing so I don't mind.
I wiped my docker containers and reinstalled.  What I was finding that was when I installed again, (at least from today) the database migration would complete successfully but after going to the last setup page, I'd land on a page that was an error, and was /setup/undefined.  Pressing on the dashboard button took me to my dashboard.
The other thing that would happen is when creating a station, I'd see an error that said could not create directories playlists, podcasts, recordings and so on.  If I made the folders myself it would let it through, but the stations number would go up in the address bar.  What I did (probably not the best idea but I wanted to see if it'd work) is install azuracast fresh, then changed the permissions on the azuracast folder so that it could make what it needed.  So far, everything seems okay.
